TIM-BC-Tool-SQCG/app/scripts/views/SegmentInfoView.js

```diff
@@ -402,7 +402,7 @@ define([
 
         if( valid ) {
           if( value > 240 ) {
-            alert('Average Incident Duration Error: Should not exceed 240.\nPrevious valid value will be reloaded.');
```

Comment:
The check condition is greater than 240. So it makes sense to say not exceed 240.

Reply:
I agree with Dan.  Please restore the original alert.  In fact, the tool tips should be say 240 as well. This applies to all modules other than SSP (SSP's logic is a bit different and Dan is / will be investigating).  I have left detailed comments on what to say in the 0428 test report.